Assets/Scripts/PickupHead.cs

```diff
@@ -5,29 +5,32 @@
 public class PickupHead : MonoBehaviour {
 
 	private GameObject body;
-	private GameObject head;
-	private HeadController headController;
+	private BodyController bodyController;
 	private Jump jumpScript;
+	private Transform parent;
 	// Use this for initialization
 	void Start () {
 		body = this.gameObject;
-		head = GameObject.Find ("Head");
-		headController = head.GetComponent<HeadController> ();
+		bodyController = body.GetComponent<BodyController> ();
+		parent = this.transform.parent;
 		jumpScript = this.GetComponent<Jump> ();
 	}
 
 	void OnCollisionEnter(Collision collision) {
 		bool isOnGround = jumpScript.GetIsOnGround();
-		if (collision.gameObject.tag == "HeadIsDetached" && isOnGround) {
+		if (collision.gameObject.tag == "HeadIsDetached" && isOnGround && bodyController.isDecapitated) {
```

Comment:
`HeadIsDetached` sounds like a flag that should be on the body. Should we rename this flag to `isBodyless` or `isSevered`?

Reply:
Ok